packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -596,6 +602,7 @@ export class GarbageCollector implements IGarbageCollector {
 
         // Whether we are running in test mode. In this mode, unreferenced nodes are immediately deleted.
         this.testMode = this.mc.config.getBoolean(gcTestModeKey) ?? this.gcOptions.runGCInTestMode === true;
+        this.tombstoneMode = this.mc.config.getBoolean(tombstoneKey) ?? this.gcOptions.tombstoneMode === true;
```

Comment:
Let's not add configs / feature flags just yet. I have an issue to do this - https://dev.azure.com/fluidframework/internal/_workitems/edit/2131. Let's do it in a separate PR as we need to have a plan of how this interacts with the current options of sweep with recovery and the future sweep options.

Reply:
I think we want it to ship dark one way or another... right?  You're just saying, you'll get to it sometime before the next minor release?

Reply:
How would I test this?

Reply:
I'm going to add this as a config so I can test the code. It'll be a hidden config

Reply:
After an offline conversation - deciding to add a temporary feature flag